doc/HowToPipeline.md

```diff
@@ -0,0 +1,2040 @@
+be done on individual detectors on a pre visit basis. This makes sense, as this is a natural
+parallelization vector. However, sometimes it is useful to consider an entire focal plane at a time, or some
+other larger scale concept, like a tract. By changing the dimensions of the unit of processing for our
+task, we by necessity change what sort of inputs our task will expect. Let's make that change now, and see
+what the consequences will be to the connection class.
+
+```python
+class ApertureTaskConnections(pipeBase.PipelineTaskConnections,
+                              defaultTemplates={"outputName":"customAperture"},
+                              dimensions=("visit", "abstract_filter", "skymap")):
+    ...
+
+    areaMasks = pipeBase.connectionTypes.PrerequisiteInput(doc="A mask of areas to be ignored",
+                                                           storageClass="Mask",
+                                                           dimensions=("visit", "detector", "abstract_filter", "skymap"),
+                                                           name="ApAreaMask",
+                                                           multiple=True)
+    exposures = pipeBase.connectionTypes.Input(doc="Input exposure to make measurements on",
+                                               dimensions=("visit", "detector", "abstract_filter", "skymap"),
+                                               storageClass="ExposureF",
+                                               name="calexp",
+                                               multiple=True)
+    inputCatalogs = pipeBase.connectionTypes.Input(doc="Input catalog with existing measurements",
+                                                   dimensions=("visit", "detector", "abstract_filter",
+                                                               "skymap"),
+                                                   storageClass="SourceCatalog",
+                                                   name="src",
+                                                   multiple=True) 
+    backgrounds = pipeBase.connectionTypes.Input(doc="Background model for the exposure",
+                                                 storageClass="Background",
+                                                 dimensions=("visit", "detector", "abstract_filter",
+                                                             "skymap"),
+                                                 name="calexpBackground",
+                                                 multiple=True)
+    ...
+```
+
+The dimensions of our `ApertureTaskConnections` class are now `visit`, `abstract_filter`, and `skymap`.
+However all of our input data-sets are themselves still defined over each of these dimensions, and also
+`detector`. That is to say you get one `calexp` for ever unique combination of `exposure`'s dimensions.
+Because the tasks's dimensions are a more inclusive set of dimensions (less specified) we expect that for a
```

Comment:
I find "more inclusive" confusing here; maybe just say "less specific" and drop the parenthetical.